org.eclipse.ice.tests.integration/org.eclipse.ice.tests.integration.commands/pom.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>org.eclipse.ice.tests.integration</artifactId>
+    <groupId>org.eclipse.ice</groupId>
+    <version>3.0.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.eclipse.ice</groupId>
+  <artifactId>org.eclipse.ice.tests.integration.commands</artifactId>
+  <version>3.0-SNAPSHOT</version>
+
+  <name>org.eclipse.ice.tests.integration.commands</name>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.11</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.ice</groupId>
+      <artifactId>org.eclipse.ice.commands</artifactId>
+      <version>0.0.1-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
+      <version>1.7.30</version>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.ice</groupId>
+      <artifactId>org.eclipse.ice.tests.data</artifactId>
+      <version>3.0.0-SNAPSHOT</version>
+    </dependency>
+    <dependency>
+      <groupId>log4j</groupId>
```

Comment:
I think we should just have one log4j dependency. I believe @dbluhm has been using `slf4j-simple` so to be consistent let's just use that one.